pyuvsim/source.py

```diff
@@ -157,15 +156,19 @@ def coherency_calc(self, telescope_location):
 
             polarized_sources = np.where(~Ionly_mask)[0]
             sinX = np.sin(self.hour_angle)
-            cosX = np.tan(telescope_location.lat) * np.cos(self.dec) - np.sin(self.dec) * np.cos(self.hour_angle)
+            cosX = np.tan(telescope_location.lat) * np.cos(self.dec) - np.sin(self.dec) * np.cos(
+                self.hour_angle)
 
-            rotation_matrix = np.array([[cosX, sinX], [-sinX, cosX]]).astype(float)       # (2, 2, Ncomponents)
+            rotation_matrix = np.array([[cosX, sinX], [-sinX, cosX]]).astype(
+                float)  # (2, 2, Ncomponents)
```

Comment:
Let's put the comment on the line before and then the code won't need to be split.

Reply:
Done.